extensions/file-cutter/src/cut-files.ts

```diff
@@ -0,0 +1,22 @@
+import { getSelectedFinderItems, showHUD, showToast, Toast } from "@raycast/api";
+import fs from "fs";
+import path from "path";
+
+const TEMP_FILE = path.join(__dirname, "cut-files.json");
```

Comment:
You can maybe check out [environment](https://developers.raycast.com/api-reference/environment) for best place to store support files